src/reconstruction/simplified_enzyme_constrained.jl

```diff
@@ -44,15 +44,22 @@ function make_simplified_enzyme_constrained_model(
     reaction_mass_groups::Maybe{Dict{String,Vector{String}}} = nothing,
     reaction_mass_group_bounds::Maybe{Dict{String,Float64}} = nothing,
 )
-    # fix kwarg inputs
+
     if !isnothing(total_reaction_mass_bound)
-        reaction_mass_groups = Dict("uncategorized" => variables(model)) # TODO should be reactions
+        (isnothing(reaction_mass_groups) && isnothing(reaction_mass_group_bounds)) ||
+            throw(
+                ArgumentError(
+                    "argument values would be overwritten by total_reaction_mass_bound!",
+                ),
+            )
+
+        reaction_mass_groups = Dict("uncategorized" => genes(model))
```

Comment:
bug spotted: why is it genes here when it is reaction_mass_group?

Reply:
(in turn, this generates a coupling matrix without entries (because nothing mathes the reaction ids))

Reply:
```suggestion
        reaction_mass_groups = Dict("uncategorized" => variables(model))
```